<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>1198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SnugQT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1258</width>
         <height>1003</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="imgLabel">
          <property name="text">
           <string>Image display</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="previousImgButton">
      <property name="toolTip">
       <string>Previous image</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
      <property name="shortcut">
       <string>Left</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="nextImgButton">
      <property name="toolTip">
       <string>Next image</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
      <property name="shortcut">
       <string>Right</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2">
     <widget class="QPlainTextEdit" name="promptLine">
      <property name="toolTip">
       <string>Enter the prompt here</string>
      </property>
      <property name="plainText">
       <string>cute rabbit on the beach</string>
      </property>
      <property name="placeholderText">
       <string>Prompt</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QPushButton" name="generateButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>64</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Generate images</string>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+Return</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" rowspan="2">
     <widget class="QPlainTextEdit" name="negPromptLine">
      <property name="toolTip">
       <string>Enter negative prompt here</string>
      </property>
      <property name="plainText">
       <string>ugly, bad quality, worst quality, frame, text, watermark</string>
      </property>
      <property name="placeholderText">
       <string>Negative prompt</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="imgDisplayIndex">
      <property name="toolTip">
       <string>Image index</string>
      </property>
      <property name="text">
       <string>----</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="promptHistoryCombo"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="negPromptHistoryCombo"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1278</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
